presto-native-execution/presto_cpp/main/PrestoServer.cpp

```diff
@@ -157,7 +158,9 @@ PrestoServer::PrestoServer(const std::string& configDirectoryPath)
 
 PrestoServer::~PrestoServer() {}
 
-void PrestoServer::run() {
+void PrestoServer::run(
+    std::function<void(proxygen::HTTPServer*)> onSuccess,
```

Comment:
How are these arguments used?

Reply:
We use it inside PrestoServerWrapper.cpp to get the socket address of the server that we need while running the test suite(see SetUpTestSuite()).

Reply:
It is not ideal to make API changes just for testing. Let's figure out another way to test.